docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -150,6 +150,79 @@ To investigate further, you can view the details of the build captured as labels
+in an observability backend alongside the traces of the pipeline builds and Jenkins health metrics.
+Storing logs in an observability backend provides several benefits including:
+
+* Better observability, monitoring, alerting, and troubleshooting of the Jenkins instance
+due to the unification of all the signals in the observability backend.
+* Better traceability and control of long term storage for builds so you can more easily
+audit the Software Delivery Lifecycle.
+* Better scalability and reliability of Jenkins by greatly reducing the quantity of data stored in
+Jenkins and limiting the well known file system performance of Jenkins when storing a large history of builds.
+
+[discrete]
+[[ci-cd-store-jenkins-logs-requirements-configuration]]
+=== Requirements and configuration
+
+Storing Jenkins pipeline logs in Elastic requires:
+
+* Elastic Observability version 8.1 or higher.
+* The OpenTelemetry Protocol endpoint configured on the Jenkins OpenTelemetry integrationmust be
+reachable from the Jenkins Agents (i.e. don't specify a localhost OTLP endpoint unless OpenTelemetry
+collectors are also deployed on the Jenkins Agents).
+* When using OpenTelemetry Collectors, set up a logs pipeline in addition to the traces and metrics pipelines.
+
+To store pipeline logs in Elastic:
+
+1. Navigate to the _OpenTelemetry_ section of Jenkins configuration screen.
+2. Set the _OTLP Endpoint_.
+3. Use the _Add Visualisation Observability Backend_ dropdown to select the *Elastic Observability* option.
+4. Set the _Kibana base URL_.
+5. Click the _Advanced_ button to choose a storage integration strategy.
+There are two options for storing pipeline logs in Elastic Observability:
+    * *Store pipeline logs in Elastic and visualize logs exclusively in Elastic*,
+    which means logs will no longer be visible through the Jenkins UI.
+    Read more in <<ci-cd-visualize-logs-kibana>> below.
+    * *Store pipeline logs in Elastic and visualize logs both in Elastic and through Jenkins*,
+    which means you can view logs stored in Elastic on demand in the Jenkins UI.
+    Read more in <<ci-cd-visualize-logs-kibana-and-jenkins>> below.
+
+[discrete]
+[[ci-cd-visualize-logs-kibana]]
+=== Visualize logs in Kibana
+
+The Jenkins OpenTelemetry provides turnkey pipeline log storage in Elasticsearch with visualization in Kibana.
+The Jenkins pipeline build console will display a hyperlink to Kibana rather than displaying the logs on the page.
+
+[role="screenshot"]
+image::images/ci-cd-visualize-logs-kibana-console.png[Jenkins Console Output page with link to view logs in Elastic Observability]
+
+.Architecture for storing pipeline logs in Elastic and visualizing logs exclusively in Elastic
+image::images/ci-cd-visualize-logs-kibana-architecture.png[Architecture diagram for storing pipeline logs in Elastic and visualizing logs exclusively in Elastic]
+
+[discrete]
+[[ci-cd-visualize-logs-kibana-and-jenkins]]
+=== Visualize in both Kibana and Jenkins
```

Comment:
```suggestion
=== Visualize logs in both Kibana and Jenkins
```

Reply:
As I added a line to recommend this setup "Visualize logs in both Kibana and Jenkins" over the setup "Visualize logs exclusively in Kibana", it may be better to switch the order in the docs and to first document "Visualize logs in both Kibana and Jenkins"